requirements/pip.txt

```diff
@@ -69,47 +69,47 @@ selectolax==0.2.1
 orjson==2.0.7  # pyup: ignore
 
 # Utils
-django-gravatar2==1.4.2
+django-gravatar2==1.4.4
 pytz==2019.3
 Unipath==1.1
 django-kombu==0.9.4
-mock==3.0.5
-stripe==2.38.0
-regex==2019.11.1
+mock==4.0.1
+stripe==2.42.0
+regex==2020.1.8
 
 # unicode-slugify==0.1.5 is not released on PyPI yet
 git+https://github.com/mozilla/unicode-slugify@b696c37#egg=unicode-slugify==0.1.5
 
-django-formtools==2.1
-django-crispy-forms==1.8.0
+django-formtools==2.2
+django-crispy-forms==1.8.1
 
-docker==4.1.0
+docker==4.2.0
 
 django-textclassifier==1.0
 # django-textclassifier doesn't have pinned versions
 # if there is an update they could break our code
 nltk==3.4.5
 textblob==0.15.3
 
-django-annoying==0.10.5
+django-annoying==0.10.6
 django-messages-extends==0.6.0
 djangorestframework-jsonp==1.0.2
-django-taggit==1.1.0
+django-taggit==1.2.0
 dj-pagination==2.4.0
 
 # Version comparison stuff
-packaging==19.2
+packaging==20.1
 
 # Commenting stuff
-django-cors-middleware==1.4.0
+django-cors-middleware==1.5.0
 
 # User agent parsing - used for analytics purposes
-user-agents==2.0
+user-agents==2.1
 
 # Utilities used to upload build media to cloud storage
-django-storages[azure]==1.7.2
-azure-storage-blob==1.5.0
-azure-storage-common==1.4.2
+django-storages[azure]==1.9.1
+azure-storage-blob==12.1.0
+azure-storage-common==2.1.0
```

Comment:
Not sure about updating this

Looks like these are handled in django-storage now https://github.com/jschneier/django-storages/pull/785

Also, see https://github.com/readthedocs/readthedocs-ext/commit/eeb6e738a1f494a530b3686274cf94864f2142e3
and https://github.com/jschneier/django-storages/issues/784